src/borg/hashindex.pyx

```diff
@@ -22,7 +22,7 @@ cdef extern from "_hashindex.c":
         uint32_t version
         char hash[16]
 
-    HashIndex *hashindex_read(object file_py, int permit_compact) except *
+    HashIndex *hashindex_read(object file_py, int permit_compact) except NULL
```

Comment:
ah, purpose is to really propagate exceptions from there (as hashindex_read calls python c api).